pkg/api/datavolume/store.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/rancher/apiserver/pkg/apierror"
 	"github.com/rancher/apiserver/pkg/types"
 	"github.com/rancher/wrangler/pkg/schemas/validation"
-	kv1alpha3 "kubevirt.io/client-go/api/v1alpha3"
+	kv "kubevirt.io/client-go/api/v1"
```

Comment:
prefer to keep the version specification with `kv1`

Reply:
Sure, I can fix that.